### DISCUSSION TITLE
¿Abandono por parte de una empresa? ¡Sigamos adelante, somos el futuro!

### DISCUSSION THREAD

**Discussion by xramons** [Launch-X-Latam/MisionFrontEnd#601] [Team Posts]
Hoy quiero hablarles acerca de una situación que seguramente todos conocen: el abandono por parte de una empresa de su compromiso con los estudiantes. En este caso, me refiero a una empresa que prometió organizar un hackathon y otorgar certificados a sus participantes, pero no cumplió con su palabra.

Además de esto, dejaron de contestar mensajes y ofrecieron excusas poco convincentes. Pero lo peor de todo, es que los ejemplos de código que presentaron simplemente no funcionaban. Esta situación puede ser frustrante para los estudiantes, especialmente aquellos que invirtieron tiempo y esfuerzo en el proyecto.

A pesar de todo esto, quiero animarlos a que sigan adelante y no se desanimen. Es importante tener en cuenta que estas experiencias son una lección valiosa para el futuro. Saber que una empresa no cumplió con su compromiso puede ser una señal de alerta cuando llegue el momento de buscar trabajo en el mercado laboral. Pero también pueden verlo como una oportunidad para demostrar que, aunque las empresas no siempre cumplen sus promesas, ustedes como estudiantes están comprometidos con su educación y su futuro.

No permitan que una mala experiencia les impida seguir adelante en su carrera. Aprendan de esto y utilicen su conocimiento y habilidades para hacer un cambio positivo en el mundo. Después de todo, ustedes son el futuro y el mundo gira gracias a su compromiso y dedicación.

Por supuesto, entiendo que muchos de ustedes pueden haber sido afectados por esta situación y pueden sentirse frustrados o desilusionados. Si desean compartir sus experiencias y opiniones sobre el tema, los invito a hacerlo en los comentarios de esta publicación. Siéntanse libres de nombrar la empresa y discutir sobre lo sucedido.

Además, me gustaría saber cómo se sienten al respecto y si en el futuro, cuando tengan que usar tecnologías de esta empresa, ¿cómo se sentirían al respecto? ¿Esto afectaría su decisión de trabajar con ellos? Es importante tener en cuenta estas cosas al considerar nuestras opciones en el mercado laboral y hacer elecciones informadas.

En resumen, no se desanimen por una mala experiencia con una empresa. Aprendan de ella y sigan adelante con su compromiso con su educación y su futuro. Y recuerden, si desean compartir sus experiencias y opiniones, ¡los espero en los comentarios!

**Comment by xramons** [comment]
Copié el post en el discussion público, dejo el link:

https://github.com/Launch-X-Latam/MisionBackend/discussions/203

**Reply by cmjosueh** [reply]
Oigan amigos encima de que es gratis, todavía escriben que esto es una mala experiencia, creo que para muchos al igual que yo el conocimiento otorgado en launch es muy valioso. ya dejen de hacerle a la mamada y mejor pónganse a repasar lo impartido en ambas misiones y traten de crear e implementar soluciones de arquitectura de desarrollo. les aseguro que muchos se quejan y ni siquiera saben implementar soluciones, en más ni programar han de saber.

**Reply by NelsonOrtizLeal** [reply]
Estoy de acuerdo, con esa opinión. El curso es gratis y el material ahí esta. Si esperas que tomando un curso ya sepas todo, así no funciona. Usa el contenido del curso como base y enfócate en lo que te llame la atención. La carrera tiene muchas áreas interesantes que puedes estudiar y especializarte.

**Comment by luiserdef** [comment]
A mi punto de vista difiero en parte con lo que has comentado, me hace percibir como que lo importante de todo esto no era el contenido que se enseñó ni de aprender sino de tener un certificado y una hackathon sin importar si realmente aprendieron durante el camino. Las personas que realmente quisieron aprender le habrán sacado provecho a las clases, aplicaron el conocimiento con las prácticas y proyectos que se dejaron. El temario que describieron tanto para frontend y backend se hizo. Para mi el objetivo que era el de enseñar lo cumplieron. Ahora, en mi caso el certificado y la hackaton me hubiera venido ni bien ni mal, perfectamente no lo hubiera tomado, para lo que vine era para aprender que cosas iban a enseñar y ya luego aplicar eso en algún proyecto que pudiera demostrar lo que sé, después de todo, no pague nada todo esto fue gratis. Lo que si no puedo ignorar es que para muchos si es posible que habrá quedado ese sabor agridulce de quedarse esperando algo que no se dió y que no se sabe si lo llegarán hacer. 
Hablar de abandono o mala experiencia por dos cosas puntuales ignorando que la idea principal que era enseñar todo el contenido lo hayan hecho, no me parece del todo acertado. ¿falta de palabra? te lo podría comprar.
No sé que pasó en realidad y me da pena que haya terminado así talvez de manera brusca y estoy seguro de que los que estuvieron enseñando o los jóvenes que presentaron el proyecto tampoco han tenido la intención de terminarlo así, antes de asumir que fueron negligentes voy a considerar todas las circunstancias que pudieron pasar antes de hacer un juicio final, problemas internos, con las empresas, imprevistos o cosas que estaban fuera de su control. Tampoco niego que me hubiera gustado un comunicado para que no quedara todo en el aire pero bueno, las cosas se dieron así.

**Comment by FedericoF03** [comment]
Para los que se están quejando, esta bien quejarse si un servicio AUNQUE GRATIS no cumple con lo dicho, no hay porque atacarlos por decir la verdad, también recuerden que no todos entraron por la misma razón algunos solo queríamos el certificado marcado por Microsoft para sumar al CV. Básicamente al no cumplir lo dicho hicieron perder el tiempo a muchas personas.
